mirgecom/simutil.py

```diff
@@ -112,8 +110,8 @@ def sim_checkpoint(discr, visualizer, eos, q, vizname, exact_soln=None,
         actx = cv.mass.array_context
         nodes = thaw(actx, discr.nodes())
         expected_state = exact_soln(x_vec=nodes, t=t, eos=eos)
-        exp_resid = q - expected_state
-        err_norms = [discr.norm(v, np.inf) for v in exp_resid]
+        exp_resid = cv - expected_state
+        err_norms = [discr.norm(v, np.inf) for v in exp_resid.join()]
         maxerr = max(err_norms)
```

Comment:
```suggestion
        maxerr = discr.norm(exp_resid, np.inf)
```

Reply:
This isn't readily apparent unless you are really looking hard, but...

the broken-out `err_norms` is reported to the user so he can track the errors in each component of the solution.  

I do see the merit in replacing the simple `max` with a norm in isolation... since we need the components anyway, however, I'm not really sure.

fyi, when I actually do the replacement `maxerr=discr.norm(exp_resid, np.inf)` didn't work.  I had to: `maxerr=discr.norm(exp_resid.join(), np.inf)`.